style/thread_state.rs

```diff
@@ -27,38 +27,34 @@ bitflags! {
     }
 }
 
-macro_rules! thread_types ( ( $( $fun:ident = $flag:path ; )* ) => (
-    impl ThreadState {
-        /// Whether the current thread is a worker thread.
-        pub fn is_worker(self) -> bool {
-            self.contains(ThreadState::IN_WORKER)
-        }
+impl ThreadState {
+    /// Whether the current thread is a worker thread.
+    pub fn is_worker(self) -> bool {
+        self.contains(ThreadState::IN_WORKER)
+    }
 
-        $(
-            #[allow(missing_docs)]
-            pub fn $fun(self) -> bool {
-                self.contains($flag)
-            }
-        )*
+    #[allow(missing_docs)]
```

Comment:
Can we get a doc comment like the one for `is_worker` for `is_script` and `is_layout`?